settings.py

```diff
@@ -24,5 +26,7 @@
 USE_TZ = False
```

Comment:
Consider setting `USE_TZ` to `True` to enable timezone support, which is generally recommended for Django projects to handle datetime objects in a consistent manner.